test/built-ins/BigInt/parseInt/S18.2.5_A2_T1.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2009 the Sputnik authors.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+info: Operator remove leading StrWhiteSpaceChar
+esid: sec-bigint-parseint-string-radix
+description: "StrWhiteSpaceChar :: TAB (U+0009)"
+---*/
+
+assert.sameValue(BigInt.parseInt("\u00091"), BigInt.parseInt("1"));
```

Comment:
why not assert the result against the actual expected value and not other parseInt call?